scripts/export.sh

```diff
@@ -30,34 +31,55 @@ nohup lily daemon --repo="${REPO_PATH}" --config=/lily/config.toml --bootstrap=f
 # Wait for Lily to come online
 lily wait-api
 
-# Extract the walking epochs
-FROM_EPOCH=$(echo "${SNAPSHOT_FILE}" | cut -d'_' -f2)
+# Extract the available walking epochs
+STATE=$(lily chain state-inspect -l 3000)
+# FROM_EPOCH=$(echo "${SNAPSHOT_FILE}" | cut -d'_' -f2)
+FROM_EPOCH=$(echo "${STATE}" | jq -r ".summary.stateroots.oldest")
+FROM_EPOCH=$((FROM_EPOCH + 1))
```

Comment:
Walking with a one epoch shift to avoid errors on the first epoch of each export.